fmriprep/workflows/bold/base.py

```diff
@@ -144,6 +144,15 @@ def init_func_preproc_wf(bold_file):
     from niworkflows.interfaces.utils import DictMerge
     from sdcflows.workflows.base import init_sdc_estimate_wf, fieldmap_wrangler
 
+    workflow = Workflow(name=wf_name)
+
+    if nb.load(bold_file).shape[3:] <= (1,):
```

Comment:
We ensure N > 5 later if STC is to be run. I'd say we can safely do that directly here.
```suggestion
    if nb.load(bold_file).shape[3:] <= (5,):
```